Demo/Demo/SwiftUIComponents/PayPalWebViews/PayPalOrderActionButton.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  PayPalOrderActionButton.swift
+//  Demo
+//
+//  Created by Victoria Park on 11/7/23.
+//
```

Comment:
```suggestion
```
Good to remove the headers.

Reply:
Oops I thought I got them all. Thanks Steven.